README.md

```diff
@@ -18,6 +18,123 @@ to facilitate unit-testing of OpenTracing Java instrumentation.
 
 Packages are deployed to Maven Central under the `io.opentracing` group.
 
+## Usage
+
+### Initialization
+
+Initialization is OpenTracing-implementation-specific. Generally speaking, the pattern is to initialize a `Tracer` once for the entire process and to use that `Tracer` for the remainder of the process lifetime. The [GlobalTracer](https://github.com/opentracing/opentracing-java/blob/master/opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java) provides a helper for singleton access to the `Tracer`.
+
+### `ActiveSpan`s, `Continuation`s, and within-process prapagation
+
+For any execution context or Thread, at most one `Span` may be "active". Of course there may be many other `Spans` involved with the execution context which are (a) started, (b) not finished, and yet (c) not "active": perhaps they are waiting for I/O, blocked on a child Span, or otherwise off the critical path.
+ 
+It's inconvenient to pass an active `Span` from function to function manually, so OpenTracing provides an `ActiveSpanSource` abstraction to provide access to an `ActiveSpan` and to pin and `capture()` it for re-activation in other execution contexts (e.g., in an async callback).
```

Comment:
I think `suspend` (with the suggested semantics) would work well in the case where the response is being handled in an async callback, because the current execution context is unlikely to be doing anything more with the span. However in a fork scenario, where multiple paths need to be passed the active span, for use as a parent, the current `capture` approach/semantics is useful.

So possible we might want to think about supporting both?

However this introduces a separate question - in the fork scenario the multiple child execution contexts potentially should only have access to the `SpanContext`, for use in creating a reference. It would then be possible to say "A Span can only be active within one execution context or Thread". I think @pavolloffay covered this scenario in his PR, dealing with the case where the `ActiveSpanSource` only had access to the `SpanContext` rather than `Span`.